amaranth-chrome-ext/src/AmaranthUtil.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Utility class with helpful functions for the Amaranth Chrome extension.
+ * All methods are static.
+ */
+class AmaranthUtil {
+  /**
+   * Removes special characters from a string.
+   * @param {string} str String to remove special characters from
+   * @param {string} filters Characters to remove from str
```

Comment:
Should this just be a class constant? Is there a use case for when you want to remove different characters?

Reply:
No, there aren't any other use cases that I know of, it was just easy to generalize. I can see how it would be simpler without the `filters` argument though, I'll remove it.

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/ab34cafb7c2dc5ed7e83d13a5c29467ecbcb32aa